prep-javascript-loops/main.js

```diff
@@ -0,0 +1,68 @@
+// create your loops here.
+function whileLoop1() {
+  const numArray = [0, 1, 2, 3, 4, 5, 6, 7, 8];
```

Comment:
this array should be empty when its defined. 
The loop should be populating it.